### PR TITLE
feat: add new regex `alphaDashes` to the list

### PR DESCRIPTION
**Pull request by Lawndlwd**
add new regex `alphaDashes` to the list of regex

Expected to use this regex for name that accept characters and dashes without space on numbers

### CODE REVIEW

File: packages/use-i18n/src/__tests__/__snapshots__/formatDate.ts.snap [hunk 1]
```diff
@@ -24,45 +24,45 @@ exports[`formatDate should work with custom format and locale es 2`] = `"jueves,
 
 exports[`formatDate should work with custom format and locale es 3`] = `"jueves, 13 F 20 d. C., 15:28:00 (tiempo universal coordinado)"`;
 
-exports[`formatDate should work with custom format and locale fr 1`] = `"jeudi 13 F 20 ap. J.-C., 16:28:00 temps universel coordonné"`;
+exports[`formatDate should work with custom format and locale fr 1`] = `"jeudi 13 F 20 ap. J.-C. 16:28:00 temps universel coordonné"`;
```
Comment (chambo-e) [inline packages/use-i18n/src/__tests__/__snapshots__/formatDate.ts.snap:27]: Why ? 🤔

Comment (philibea) [reply packages/use-i18n/src/__tests__/__snapshots__/formatDate.ts.snap:27]: node 18...

Comment (philibea) [reply packages/use-i18n/src/__tests__/__snapshots__/formatDate.ts.snap:27]: But this should be updated in this PR ? @Lawndlwd

Comment (Lawndlwd) [reply packages/use-i18n/src/__tests__/__snapshots__/formatDate.ts.snap:27]: i did` pnpm test` it fails on almost all packages, so it seems it needed snapshot update

Comment (Lawndlwd) [reply packages/use-i18n/src/__tests__/__snapshots__/formatDate.ts.snap:27]: what i need to do more than `rm -rf node module && pnpm install` to update my pnpm-lock ?

**Review by philibea — APPROVED** [review]

**Review by matthprost — APPROVED** [review]

**Pull request merged**